exporter/coralogixexporter/README.md

```diff
@@ -80,24 +80,25 @@ Depending on your region, you might need to use a different domain. Here are the
 
 | Region  | Domain                        |
 |---------|---------------------------------|
-| USA1    | `coralogix.us`      |
-| APAC1   | `coralogix.in`      |
-| APAC2   | `coralogixsg.com`   |
-| EUROPE1 | `coralogix.com`     |
-| EUROPE2 | `eu2.coralogix.com` |
+| USA1    | `coralogix.us`        |
+| APAC1   | `coralogix.in`        |
+| APAC2   | `coralogixsg.com`     |
+| EUROPE1 | `coralogix.com`       |
+| EUROPE2 | `eu2.coralogix.com`   |
+| USA2    | `cx498.coralogix.com` |
```

Comment:
nit: Would it make sense to move this on the line after `USA1` so we have all related regions together?